arxiv/base/templates/base/macros.html

```diff
@@ -170,7 +170,7 @@ <h1 class="title mathjax"><span class="descriptor">Title:</span>{{ title|tex2utf
         {%- if doi %}
         <tr>
           <td class="tablecell label"><abbr title="Digital Object Identifier">DOI</abbr>:</td>
-          <td class="tablecell msc_classes"><a href="{{ doi|url_for_doi|clickthrough_url }}">{{ doi }}</a></td>
+          <td class="tablecell msc_classes"><a href="{{ doi| urlize }}">{{ doi }}</a></td>
```

Comment:
very minor: whitespace to right of pipe

```suggestion
          <td class="tablecell msc_classes"><a href="{{ doi|urlize }}">{{ doi }}</a></td>
```

Reply:
thanks, good catch!